Description: always use system libraries and remov the dependency to mini_portile
Author: Jérémy Bobbio <lunar@debian.org>
Origin: vendor
Forwarded: not-needed

--- a/Rakefile
+++ b/Rakefile
@@ -42,10 +42,6 @@
     # GENERATED_TOKENIZER
   ]
 
-  self.extra_deps += [
-    ["mini_portile",    "~> 0.5.0"],
-  ]
-
   self.extra_dev_deps += [
     ["hoe-bundler",     ">= 1.1"],
     ["hoe-debugging",   ">= 1.0.3"],
--- a/ext/nokogiri/extconf.rb
+++ b/ext/nokogiri/extconf.rb
@@ -45,7 +45,7 @@
   XML2_HEADER_DIRS = [File.join(INCLUDEDIR, "libxml2"), INCLUDEDIR]
 
 else
-  if ENV['NOKOGIRI_USE_SYSTEM_LIBRARIES']
+  if ENV['NOKOGIRI_USE_SYSTEM_LIBRARIES'] || true
     HEADER_DIRS = [
       # First search /opt/local for macports
       '/opt/local/include',
--- a/metadata.yml
+++ b/metadata.yml
@@ -15,16 +15,6 @@
 date: 2013-12-14 00:00:00 Z
 dependencies: 
 - !ruby/object:Gem::Dependency 
-  version_requirements: &id001 !ruby/object:Gem::Requirement 
-    requirements: 
-    - - ~>
-      - !ruby/object:Gem::Version 
-        version: 0.5.0
-  type: :runtime
-  name: mini_portile
-  prerelease: false
-  requirement: *id001
-- !ruby/object:Gem::Dependency 
   version_requirements: &id002 !ruby/object:Gem::Requirement 
     requirements: 
     - - ~>
